app/controllers/api/account.php

```diff
@@ -501,6 +508,7 @@
                         'email' => $email,
                         'emailVerification' => true,
                         'status' => true, // Email should already be authenticated by OAuth2 provider
+                        'passwordHistory' => $passwordHistory > 0 ? [Auth::passwordHash(Auth::passwordGenerator(), Auth::DEFAULT_ALGO, Auth::DEFAULT_ALGO_OPTIONS)] : null,
```

Comment:
Let's make a variable to prevent code diplication (passwordHistory, password)

Reply:
updated